lib/plugins/aws/lib/naming.js

```diff
@@ -332,11 +335,20 @@ module.exports = {
   },
 
   // Schedule
+  normalizeScheduleName(scheduleName) {
```

Comment:
Let's not created thise utils, but use `normalizeNameToAlphaNumericOnly` instead. This is what we use in all other similar cases.

Reply:
As per suggestion, using the `normalizeNameToAlphanumericOnly` would convert `your-scheduled-event-name` into `Yourscheduledeventname`. Would that be acceptable?

Reply:
Yes, we use it for other similar cases, so I believe we should not introduce exception here.

I personally also would prefer convertion to camelCase, but that should probably be done globally for all cases and possibly with major bump